### PR TITLE
:robot: Add winscp.vm

### PR DESCRIPTION
**Pull request by vm-packages**
Automated PR from https://github.com/mandiant/VM-Packages/issues/563 :rocket:

### CODE REVIEW

File: packages/winscp.vm/tools/chocolateyinstall.ps1 [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+  $toolName = 'WinSCP'
+  $category = 'Utilities'
+  $shimPath = '\bin\winscp.exe'
+
+  $shortcutDir = Join-Path ${Env:TOOL_LIST_DIR} $category
+  $shortcut = Join-Path $shortcutDir "$toolName.lnk"
+  $executablePath = Join-Path ${Env:ChocolateyInstall} $shimPath -Resolve
```
Comment (Ana06) [inline packages/winscp.vm/tools/chocolateyinstall.ps1:11]: Should we add the tool to path as well?
```
Install-BinFile -Name $toolName -Path $executablePath
```

File: packages/winscp.vm/tools/chocolateyinstall.ps1 [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  Install-ChocolateyShortcut -shortcutFilePath $shortcut -targetPath $executablePath -RunAsAdmin
+  VM-Assert-Path $shortcut
+
+  # Delete Desktop shortcut
+  $desktopShortcut = Join-Path ${Env:Public} "Desktop\$toolName.lnk"
+  if (Test-Path $desktopShortcut) {
```
Comment (Ana06) [inline packages/winscp.vm/tools/chocolateyinstall.ps1:17]: [nitpick] This `if` is not needed as the next command has the `-ea 0`that is the same as `-ErrorAction SilentlyContinue`. We have currently some packages with this if and some without. I would remove it for simplicity.

**Comment by nos3curity** [comment]
Addressed both issues. Ready for review.

**Review by Menn1s — APPROVED** [review]
Looks good after the white space removal

**Pull request merged**